typescript/cli/examples/submit/strategy/json-rpc-timelock-strategy.yaml

```diff
@@ -0,0 +1,17 @@
+anvil3:
+  submitter:
+    type: timelockController
+    chain: anvil3
+    # Address of the deployed timelock
+    timelockAddress: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266'
+    # Optional, the salt to be used for computing the operation id
```

Comment:
how come these are optional? aren't they required in order to generate the id?

Reply:
By default they are set to the 0 hash if not provided